explainers/subresource-loading.md

```diff
@@ -253,6 +275,24 @@ compressed](https://github.com/yoavweiss/url_compression_experiments).
 
 ## Alternate designs
 
+### `<link>`-based API
+
+This explainer had used `<link>`-based API before adopting `<script>`-based API:
+
+```html
+<link
+  rel="webbundle"
+  href="https://example.com/dir/subresources.wbn"
+  resources="https://example.com/dir/a.js https://example.com/dir/b.js https://example.com/dir/c.png"
+/>
+```
+
+However, we abandoned `<link>`-based API, in favor of `<script>`-based
+API. See [issue #580](https://github.com/WICG/webpackage/issues/580)
+for the motivation. Note that some of the following alternate designs
+were proposed at the era of `<link>`-based API. This explainer doesn't
+rewrite them with `<script>`-based API yet.
+
```

Comment:
How about adding the Chromium's implementation status here?

```
Note that Chromium's experimental implementation currently supports only
`<link>`-based API as of M95.
```

Reply:
Sure. Done.